py/desispec/fluxcalibration.py

```diff
@@ -175,7 +175,7 @@ def findappMag(flux,wave,filt):
         #Normalizing using only SDSS_R band magnitude
         if v.upper() == 'SDSS_R' or v.upper() =='DECAM_R' :
             refmag=mags[i]
-            filter_response=read_filter_response(v,basepath) # outputs wavelength,qe
+            filter_response=load_filter_response(v) # outputs wavelength,qe
             rebinned_model_flux=rebinSpectra(stdflux,stdwave,filter_response[0])
             apMag=findappMag(rebinned_model_flux,filter_response[0],filter_response[1])
```

Comment:
Is there a good reason not to use the convolution method implemented by the returned `filter_response` object here?  In other words, replace these two lines with:
```
apMag = filter_response.get_ab_magnitude(stdflux, stdwave)
```